modules/virt-adding-a-boot-source-web.adoc

```diff
@@ -29,31 +29,20 @@ There are four methods for selecting and adding a boot source in the web console
 .Procedure
 
 . In the {product-title} console, click *Virtualization* -> *Templates* from the side menu.
-
-. Click *Add source* for the template that you want to configure.
-
-. In the *Add boot source to template* window, click *Select boot source*, select a method for creating a persistent volume claim (PVC): *Upload local file*, *URL*, *Clone*, or *Registry*.
-
-. Optional: Click *This is a CD-ROM boot source* to mount a CD-ROM and use it to install the operating system on to an empty disk. The additional empty disk is automatically created and mounted by {VirtProductName}. If the additional disk is not needed, you can remove it when you create the virtual machine.
-
+. Click the options menu beside a template and select *Edit boot source*.
+. Click *Add disk*.
+. In the *Add disk* window, select *Use this disk as a boot source*.
+. Enter the disk name and select a *Source*, for example, *Blank (creates PVC)* or *Use an existing PVC*.
 . Enter a value for *Persistent Volume Claim size* to specify the PVC size that is adequate for the uncompressed image and any additional space that is required.
-
-.. Optional: Enter a name for *Boot source* to associate the name with this template.
-
-.. Optional: *Advanced Storage settings*: Click *Storage class* and select the storage class that is used to create the disk. Typically, this storage class is the default storage class that is created for use by all PVCs.
+. Select a *Type*, for example, *Disk*, which is required for hot-plugging, or *CD-ROM*.
```

Comment:
This has a ton of commas in it. Just wondering if it could be rewritten to reduce them. Maybe the part about Disk being required for hot plugging could be separate, if it is important to mention.